src/page/NewCheck/__tests__/ApiEndPointChecks/CommonFields.payload.test.tsx

```diff
@@ -77,6 +77,7 @@ describe('Api endpoint checks - common fields payload', () => {
         });
       });
 
+      // TODO: Remove/keep? It's a dubplicate of the one in `4-alerting.payload.test.tsx`
```

Comment:
Keep this one and remove the multi step one.

Reply:
I noticed a relevant difference `checkType`.
I just removed the comment.